ReadMe.md

```diff
@@ -1,71 +1,74 @@
 # DITTO-UI
 
-Easy to use web interface for biologists to look for likely pathogenic variants and understand their deleteriousness
+Easy to use web interface for biologists to look for genetic variants and understand their deleteriousness
 using DITTO scores.
 
 _!!! For research purposes only !!!_
 
 ## Description
 
-A web app where anyone can lookup variants and understand the mechanism/details of these variants such as domain,
-function, DITTO deleterious score and Clinvar reported significance.
+A web app where one can lookup variants and understand the deleteriousness using DITTO deleterious score and Clinvar
+reported significance. DITTO uses an explainable neural network model to predict the functional impact of variants and
+utilizes SHAP to explain the model's predictions. It is trained on variants from ClinVar and uses OpenCravat for
+annotations from various data sources. The higher the score, the more likely the variant is deleterious.
 
 ## Data
 
-DITTO-UI comprises precomputed DITTO scores for missense variants using dbNSFP data (downloaded on 03/19/2022) for
-training DITTO and making predictions. Domain information is extracted from Uniprot via an API query on page render.
+DITTO-UI comprises DITTO scores for any variant using annotations from openCravat
+for making predictions. Annotations are extracted from openCravat API query on page render.
 
 ## Usage
 
 DITTO-UI is deployed on the Streamlit Cloud: [DITTO-UI site](https://cgds-ditto.streamlit.app).
 
-### Local Installation and Setup
+### Installation
 
 Installation simply requires fetching the source code. Following are required:
 
--   Git (version 2+)
--   Anaconda3 (a.k.a. Conda) or the drop-in replacement Mamba (a reimplementation of Conda, tested on version 0.25 and
-    1.1.0, [install instructions](https://mamba.readthedocs.io/en/latest/installation.html))
+- Git
 
 To fetch source code, change in to directory of your choice and run:
 
 ```sh
-git clone https://github.com/uab-cgds-worthey/DITTO-UI.git
+    git clone https://github.com/uab-cgds-worthey/DITTO-UI.git
 ```
 
-#### Environement Creation
+### Requirements
+
+*OS:*
+
+Currently works only in Mac OS. Docker versions may need to be explored later to make it useable in Mac (and
```

Comment:
>useable in Mac

Did you mean to say linux? Btw, I'm surprised it wouldn't work in both Mac and Linux.

Reply:
Edited it but I'm not sure.

Reply:
If you are not certain if it works in other OS, just say it was developed/tested in Mac. This way, we don't need to assume anything.